src/Optimizely/Optimizely.php

```diff
@@ -464,6 +494,17 @@ public function isFeatureEnabled($featureFlagKey, $userId, $attributes = null)
         }
 
         $this->_logger->log(Logger::INFO, "Feature Flag '{$featureFlagKey}' is enabled for user '{$userId}'.");
+
```

Comment:
why event is not fired.

Reply:
Will do it today. Needs a new method in projectconfig to get variation out of rollout experiment